inc/class-cli-command.php

```diff
@@ -78,6 +78,36 @@ public function delete( $args, $assoc_args ) {
 			}
 		}
 	}
+
+	/**
+	 * Set id as primary key in the Native PHP Sessions plugin table.
+	 *
+	 * @subcommand add-index
+	 */
+	public function add_index( $args, $assoc_arc ) {
+		$pan_session = new \Pantheon_Sessions();
```

Comment:
Nit: I don't see a specific reason why this variable needs to be shortened to `pan_`. `pantheon_` would be sufficient. 🔢